tests/bluetooth/tester/src/btp_micp.c

```diff
@@ -0,0 +1,275 @@
+/* btp_micp.c - Bluetooth MICP Tester */
+
+/*
+ * Copyright (c) 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stddef.h>
+#include <errno.h>
+
+#include <zephyr/types.h>
+#include <zephyr/kernel.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/audio/audio.h>
+#include <zephyr/bluetooth/audio/micp.h>
+#include <zephyr/bluetooth/audio/aics.h>
+
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/byteorder.h>
+
+#include <../../subsys/bluetooth/audio/micp_internal.h>
+#include <../../subsys/bluetooth/audio/aics_internal.h>
+
+#include "bap_endpoint.h"
+#include "btp/btp.h"
+
+#define LOG_MODULE_NAME bttester_micp
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
+
+static struct bt_micp_mic_ctlr *mic_ctlr;
+
+#if defined(CONFIG_BT_MICP_MIC_CTLR_AICS)
+static struct bt_micp_included micp_included;
+struct chrc_handles {
+	uint16_t mute_handle;
+	uint16_t state_handle;
+	uint16_t gain_handle;
+	uint16_t type_handle;
+	uint16_t status_handle;
+	uint16_t control_handle;
+	uint16_t desc_handle;
+};
+struct chrc_handles micp_handles;
+extern struct btp_aics_instance aics_client_instance;
+extern struct bt_aics_cb aics_client_cb;
+#endif /* CONFIG_BT_MICP_MIC_CTLR_AICS */
+
+static void btp_send_micp_found_ev(struct bt_conn *conn, const struct chrc_handles *micp_handles)
+{
+	struct btp_micp_discovered_ev ev;
+	struct bt_conn_info info;
+
+	(void)bt_conn_get_info(conn, &info);
+	bt_addr_le_copy(&ev.address, info.le.dst);
+
+	ev.mute_handle = sys_cpu_to_le16(micp_handles->mute_handle);
+	ev.state_handle = sys_cpu_to_le16(micp_handles->state_handle);
+	ev.gain_handle = sys_cpu_to_le16(micp_handles->gain_handle);
+	ev.type_handle = sys_cpu_to_le16(micp_handles->type_handle);
+	ev.status_handle = sys_cpu_to_le16(micp_handles->status_handle);
+	ev.control_handle = sys_cpu_to_le16(micp_handles->control_handle);
+	ev.desc_handle = sys_cpu_to_le16(micp_handles->desc_handle);
+
+	tester_event(BTP_SERVICE_ID_MICP, BTP_MICP_DISCOVERED_EV, &ev, sizeof(ev));
+}
+
+static void btp_send_micp_mute_state_ev(struct bt_conn *conn, uint8_t mute)
+{
+	struct btp_micp_mute_state_ev ev;
+	struct bt_conn_info info;
+
+	(void)bt_conn_get_info(conn, &info);
+	bt_addr_le_copy(&ev.address, info.le.dst);
+
+	ev.mute = mute;
+
+	tester_event(BTP_SERVICE_ID_MICP, BTP_MICP_MUTE_STATE_EV, &ev, sizeof(ev));
+}
+
+static void micp_mic_ctlr_mute_cb(struct bt_micp_mic_ctlr *mic_ctlr, int err,
+				  uint8_t mute)
```

Comment:
```suggestion
static void micp_mic_ctlr_mute_cb(struct bt_micp_mic_ctlr *mic_ctlr, int err, uint8_t mute)
```

Maximum line length is 100, suggest to use that to that